docs/src/dedication/architecture/deploying.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Deployment"
+weight: 3
+sidebarTitle: "Deploying"
+---
+
+## Deploying to Production and Staging
+
+The `production` branch of your Git repository is designated for production, and a `staging` branch is designated for staging.  Any code merged to those branches will automatically trigger a rebuild of the production and staging environments, respectively, in the Dedicated Cluster.  Any defined users or environment variables will also be propagated to the Dedicated Cluster as well.
```

Comment:
This is true for the Old Dedicated, not for the new Dedication.  In the new setup, `master` is production just like on a normal Grid project.